app/presenters/organisations/contacts_presenter.rb

```diff
@@ -32,6 +32,7 @@ def foi_previous_releases_link
     def format_contacts(contacts, foi: false)
       contacts.map do |contact|
         {
+          locale: contact["locale"] || "",
```

Comment:
We could use a neat Rails helper here - `presence`. This returns the content of `contact["locale"]` unless it's blank / not there when it returns `nil`.

```suggestion
          locale: contact["locale"].presence,
```